templates/views.py

```diff
@@ -0,0 +1,75 @@
+from venv import create
```

Comment:
I don't seem to understand why views.py will be inside a templates folder.

Please review these changes and notify me after making changes or with reasons why you think it should be here.